src/Exceptions/PasswordDoesMatchException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Exceptions;
+
+use Exception;
+
+class PasswordDoesMatchException extends Exception
+{
+    public function __construct()
```

Comment:
That's a bad practice. PHP Exception class has more complex constructor with default values and message is just first of them. You shouldn't change constructor signature without any reason.